hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/MembershipManager.java

```diff
@@ -752,9 +753,12 @@ private void removeMember(MemberImpl member, String reason, boolean shouldCloseC
     }
 
     private void closeConnection(Address address, String reason) {
-        Connection conn = node.getServer().getConnectionManager(MEMBER).get(address);
-        if (conn != null) {
-            conn.close(reason, null);
+        int planeCount = node.getServer().getContext().properties().getInteger(CHANNEL_COUNT);
```

Comment:
Since we usually do close action over connection instead of `ServerConnectionManager`, I introduced a `getAllConnections(Address)` in the connection manager and closed the connections over the connections I obtained